omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -103,9 +103,31 @@
 			</tr>
 		</table>
 	</fieldset>	
-	
 	<br/>
 	
+    <fieldset>
+        <legend><openmrs:message code="legacyui.manageuser.providerAccount"/></legend>
+        <c:if test="${isProvider}">
+            <openmrs:message code="legacyui.manageuser.providerIdentfier"/>
+            <c:forEach items="${providerList}" var="currentProvider" varStatus="loop">   
+                <c:if test="${empty currentProvider.identifier}">
```

Comment:
`empty` checks for null values and prints the value accordingly. I believe this is how you wanted it done in the JSP :)